tests/kernels/graph_kernels/labelled_random_walk.py

```diff
@@ -0,0 +1,127 @@
+"""
```

Comment:
I guess we can rename these folders and files as test_graph_kernels, test_labelled_random_walk etc. I think this might already have been done, not sure!